src/components/UI/Carousel/Carousel.module.css

```diff
@@ -0,0 +1,63 @@
+.carousel-container {
+  padding: 2rem;
+}
+
+.carousel {
+  position: relative;
+}
+
+.carousel-row {
+  display: flex;
+  flex-flow: row;
+  gap: 1rem;
+  overflow-x: scroll;
+}
+
+.carousel-row::-webkit-scrollbar {
+  display: none;
+}
+
+.left-arrow {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: max-content;
+  background-color: var(--COLOR-bg);
+  padding: 10px;
+  position: absolute;
+  left: 0;
+  opacity: 0;
+}
+
+.right-arrow {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: max-content;
+  background-color: var(--COLOR-bg);
+  padding: 10px;
+  position: absolute;
+  right: 0;
+  opacity: 0;
```

Comment:
Suggestion: you could combine common properties like `display `and `align-items` to avoid repetition.

for-example: 
.right-arrow, .left-arrow {
    display: flex;
    justify-content: center;
    align-items: center;
    width: max-content;
    .....
}